### PR TITLE
NB-101 This is my story description

### PR DESCRIPTION
**Pull request by KatyTaylor**
I added a line to the protocol for test reasons

General useful stuff for reviewers to know

### CODE REVIEW

File: NewBank/protocol.txt [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+This document details the protocol for interacting with the NewBank server.
 
-A customer enters the command below and sees the messages returned 
+A customer enters the command below and sees the messages returned
 
 SHOWMYACCOUNTS
-Returns a list of all the customers accounts along with their current balance 
-e.g. Main: 1000.0 
+Returns a list of all the customers accounts along with their current balance
+e.g. Main: 1000.0
```
Comment (KatyTaylor) [inline NewBank/protocol.txt:7]: why is there a change here?

**Review by leigh712 — APPROVED** [review]
looks good :)

File: NewBank/protocol.txt [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 
 NEWACCOUNT <Name>
 e.g. NEWACCOUNT Savings
 Returns SUCCESS or FAIL
 
 MOVE <Amount> <From> <To>
-e.g. MOVE 100 Main Savings 
+e.g. MOVE 100 Main Savings
```
Comment (KatyTaylor) [inline NewBank/protocol.txt:14]: this looks great!

File: NewBank/protocol.txt [hunk 3]
```diff
@@ -1,19 +1,22 @@
-This document details the protocol for interacting with the NewBank server.  
+This document details the protocol for interacting with the NewBank server.
 
-A customer enters the command below and sees the messages returned 
+A customer enters the command below and sees the messages returned
```
Comment (KatyTaylor) [inline NewBank/protocol.txt:3]: example

**Comment by KatyTaylor** [comment]
This wasn't a real pull request

**Pull request closed without merging**